core/embed/extmod/modtrezorcrypto/modtrezorcrypto-ed25519.h

```diff
@@ -100,6 +100,8 @@ STATIC mp_obj_t mod_trezorcrypto_ed25519_sign(size_t n_args,
 STATIC MP_DEFINE_CONST_FUN_OBJ_VAR_BETWEEN(mod_trezorcrypto_ed25519_sign_obj, 2,
                                            3, mod_trezorcrypto_ed25519_sign);
 
+#if !BITCOIN_ONLY
+
```

Comment:
Shouldn't we remove all ed25519-related functions?

Reply:
Same reason as below.